db/table_cache.cc

```diff
@@ -336,10 +337,21 @@ Status TableCache::Get(const ReadOptions& options,
 
     if (auto row_handle =
             ioptions_.row_cache->Lookup(row_cache_key.GetUserKey())) {
+
+      // Cleanable routine to release the cache entry
+      Cleanable cache_entry_clean;
+      auto release_cache_entry_func = [] (void* cache_to_clean, void* cache_handle){
+          ((Cache*)cache_to_clean)->Release((Cache::Handle*) cache_handle);
+      };
       auto found_row_cache_entry = static_cast<const std::string*>(
           ioptions_.row_cache->Value(row_handle));
-      replayGetContextLog(*found_row_cache_entry, user_key, get_context);
-      ioptions_.row_cache->Release(row_handle);
+      // If it comes here value is located on the cache, input PinnableSlice (get_context.pinnable_slice_) will now
```

Comment:
You wanted to rename cache_entry_clean to value_pinner.

Nits:
After replayGetContextLog() returns => After replayGetContextLog() returns,
((or a => (or a
is reset => is reset.